Tests/SentryTests/Helper/SentryThreadWrapperTests.swift

```diff
@@ -0,0 +1,25 @@
+import XCTest
+
+final class SentryThreadWrapperTests: XCTestCase {
+    func testOnMainThreadFromMainThread() {
+        let e = expectation(description: "Asserted that execution happened on main thread")
+        SentryThreadWrapper.onMainThread {
+            XCTAssertTrue(Thread.isMainThread)
+            e.fulfill()
+        }
+        waitForExpectations(timeout: 1)
+    }
+
+    func testOnMainThreadFromNonmainContext() {
```

Comment:
`l`: Shouldn't it be `NonMain`?
```suggestion
    func testOnMainThreadFromNonMainContext() {
```